content/cohorts/ds-ai/ottaa-project.en.adoc

```diff
@@ -0,0 +1,34 @@
+---
+title: OTTAA Project
+weight: 50
+categories: "team"
+tags: ["team"]
+type: profiles
+
+description: An Alternative Augmentative Communication (AAC) system, intended for people with speech disabilities.
+country: Chile
+date: 2020-09-25
+github: https://github.com/OTTAA-Project
+linkedin: https://www.linkedin.com/company/ottaa-project/
+twitter: https://twitter.com/OTTAAProject
```

Comment:
We are very strong on Facebook and Instagram, would be nice to have those handlers as well. 
Both are @OTTAAProject